src/Custom/Common/RequestOptionsExtensions.cs

```diff
@@ -0,0 +1,22 @@
+using System.ClientModel.Primitives;
+using System.Threading;
+
+internal static class RequestOptionsExtensions
```

Comment:
Minor: should this be `CancellationTokenExtensions` given that's the `this` type being extended?